artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/messages/DestAbstract.java

```diff
@@ -137,4 +141,16 @@ public DestAbstract setSerializer(String serializer) {
       this.serializer = serializer;
       return this;
    }
+
+   @Override
+   public Object execute(ActionContext context) throws Exception {
+      super.execute(context);
+
+      if (oldBatchSize > 0) {
+         context.out.println("--txt-size is deprecated, please use --commit-interval");
+         txBatchSize = oldBatchSize;
```

Comment:
If both were somehow set, I would expect the new one to always win...here the old one will win.